Keas.Mvc/Controllers/Api/AccessController.cs

```diff
@@ -37,17 +37,18 @@ public string GetTeam()
         }
 
         [HttpGet]
+        [ProducesResponseType(typeof(Access), StatusCodes.Status200OK)]
```

Comment:
Any method that returns a `List<T>` or `Array<T>` should specify `IEnumerable<T>` as the return type. So in this case it would be `[ProducesResponseType(typeof(IEnumerable<Access>), StatusCodes.Status200OK)]`

Pretty sure there are a couple other methods in this file that will need this pattern applied.